.remarkrc

```diff
@@ -23,7 +23,7 @@
       "list-item-indent": "space",
       "list-item-spacing": false,
       "maximum-heading-length": 60,
-      "maximum-line-length": 80,
+      "maximum-line-length": 120,
```

Comment:
Is this needed in this PR?
I can't see any other change here that requires this one, there are no changes in the Markdown files in this PR.
So — why this change?